.github/workflows/build_and_test.yml

```diff
@@ -366,9 +366,7 @@ jobs:
         #   See also https://github.com/sphinx-doc/sphinx/issues/7551.
         # Jinja2 3.0.0+ causes error when building with Sphinx.
         #   See also https://issues.apache.org/jira/browse/SPARK-35375.
-        # TODO(SPARK-35684, SPARK-35683): Bump up the mypy version. This is blocked by
-        #   Index.difference logic issue.
-        python3.6 -m pip install flake8 pydata_sphinx_theme 'mypy==0.812' numpydoc 'jinja2<3.0.0' 'black==21.5b2'
+        python3.6 -m pip install flake8 pydata_sphinx_theme 'mypy==0.910' numpydoc 'jinja2<3.0.0' 'black==21.5b2' types-tabulate
```

Comment:
I'd rather add the `tabulate` entry in `mypy.ini`.

```
[mypy-tabulate.*]
ignore_missing_imports = True
```

Reply:
Or we should install other libraries and remove the entries in `mypy.ini` (in the future)?

Reply:
Oh okay. I think missing import would be better approach